databricks/koalas/series.py

```diff
@@ -3184,6 +3184,190 @@ def copy(self) -> 'Series':
+
+        >>> midx = pd.MultiIndex([['lama', 'cow', 'falcon'],
+        ...                       ['speed', 'weight', 'length']],
+        ...                      [[0, 0, 0, 1, 1, 1, 2, 2, 2],
+        ...                       [0, 1, 2, 0, 1, 2, 0, 1, 2]])
+        >>> s = ks.Series([45, 200, 1.2, 30, 250, 1.5, 320, 1, 0.3],
+        ...               index=midx)
+        >>> s
+        lama    speed      45.0
+                weight    200.0
+                length      1.2
+        cow     speed      30.0
+                weight    250.0
+                length      1.5
+        falcon  speed     320.0
+                weight      1.0
+                length      0.3
+        Name: 0, dtype: float64
+
+        >>> s.replace(45, 450)
+        lama    speed     450.0
+                weight    200.0
+                length      1.2
+        cow     speed      30.0
+                weight    250.0
+                length      1.5
+        falcon  speed     320.0
+                weight      1.0
+                length      0.3
+        Name: 0, dtype: float64
+
+        >>> s.replace([45, 30, 320], 500)
+        lama    speed     500.0
+                weight    200.0
+                length      1.2
+        cow     speed     500.0
+                weight    250.0
+                length      1.5
+        falcon  speed     500.0
+                weight      1.0
+                length      0.3
+        Name: 0, dtype: float64
+
+        >>> s.replace({45: 450, 30: 300})
+        lama    speed     450.0
+                weight    200.0
+                length      1.2
+        cow     speed     300.0
+                weight    250.0
+                length      1.5
+        falcon  speed     320.0
+                weight      1.0
+                length      0.3
+        Name: 0, dtype: float64
+        """
+        if not isinstance(to_replace, (str, list, dict, int, float)):
+            raise ValueError(
+                "'to_replace' should be one of str, list, dict, int, float")
+        if to_replace is None:
+            return self
+        if regex:
+            raise NotImplementedError("replace currently not support for regex")
+        if isinstance(to_replace, list) and isinstance(value, list):
+            if not len(to_replace) == len(value):
+                raise ValueError("Replacement lists must match in length. Expecting {} got {}"
+                                 .format(len(to_replace), len(value)))
+            to_replace = {k: v for k, v in zip(to_replace, value)}
+        if isinstance(to_replace, dict):
+            is_start = True
+            for to_replace_, value in to_replace.items():
+                if is_start:
+                    current = F.when(self._scol == F.lit(to_replace_), value)
+                    is_start = False
+                else:
+                    current = current.when(self._scol == F.lit(to_replace_), value)
+            current = current.otherwise(self._scol)
```

Comment:
What if the `to_replace` is empty?

Reply:
oops, i missed that case.
just fixed it. Thanks !! :)